pkg/webui/lib/components/error-message/index.js

```diff
@@ -19,7 +19,7 @@ import Message from '@ttn-lw/lib/components/message'
 import PropTypes from '@ttn-lw/lib/prop-types'
 import { hasCauses, getBackendErrorRootCause, toMessageProps } from '@ttn-lw/lib/errors/utils'
 
-const ErrorMessage = function({ content, withRootCause, className, ...rest }) {
+const ErrorMessage = ({ content, withRootCause, className, ...rest }) => {
   const baseProps = {
```

Comment:
Please also add a check here whether the content is a backend error message and if yes, apply the `convertBackticks` props to the `<Message />` as well.